PlexRequests.UI/Modules/UserLoginModule.cs

```diff
@@ -98,9 +100,17 @@ private Response LoginUser()
                 var signedIn = (PlexAuthentication)Api.SignIn(username, password);
                 if (signedIn.user?.authentication_token != null)
                 {
-                    Log.Debug("Correct credentials, checking if the user is in the friends list");
-                    authenticated = CheckIfUserIsInPlexFriends(username, settings.PlexAuthToken);
-                    Log.Debug("Friends list result = {0}", authenticated);
+                    Log.Debug("Correct credentials, checking if the user is account owner or in the friends list");
+                    if (CheckIfUserIsOwner(settings.PlexAuthToken, username))
+                    {
+                        Log.Debug("User is the account owner");
+                        authenticated = true;
+                    }
+                    else
+                    {
+                        authenticated = CheckIfUserIsInPlexFriends(username, settings.PlexAuthToken);
+                        Log.Debug("Friends list result = {0}", authenticated);
+                    }
                 }
             }
             else if(settings.UserAuthentication) // Check against the users in Plex

```

Comment:
We also need to check if it's an owner in this case, where we don't have the password enabled but we do want to authenticate